docs/Custom-Brimcap-Config.md

```diff
@@ -28,9 +28,8 @@ choosing. This article includes examples of both configurations along with
 
 The goal in our first example customization will be to run Zui with the latest
 GA binary releases of [Zeek](https://github.com/zeek/zeek/wiki/Binary-Packages)
-and [Suricata](https://suricata.readthedocs.io/en/latest/install.html#install-binary-packages),
-as these are newer than the versions that currently ship with Brimcap. We'll
-use Linux Ubuntu 20.04 as our OS platform. On such a host, the following
+and [Suricata](https://suricata.readthedocs.io/en/latest/install.html#install-binary-packages).
+We'll use Linux Ubuntu 20.04 as our OS platform. On such a host, the following
```

Comment:
Strictly speaking, we can't say that GA releases of _both_ Zeek and Suricata are newer than the ones with Brimcap, since we're now up-to-date on Zeek. Therefore I dropped that part of the text.